src/main/java/com/github/sgov/server/service/WorkspacePublicationService.java

```diff
@@ -102,14 +100,15 @@ public URI publish(URI workspaceUri) {
     private void publishVocabularyContexts(Git git, File dir, Workspace workspace) {
         // changes might come from tools
         repositoryService.flush();
-        workspace.getAttachmentContexts().forEach(ac -> {
+        workspace.getAllAttachmentContexts().forEach(ac -> {
             if (ac.getBasedOnVersion() == null) {
-                String uriPath = ac.getUri().getPath();
-                String uuid = uriPath.substring(uriPath.lastIndexOf("/") + 1);
-                ac.setBasedOnVersion(URI.create(Vocabulary.s_c_priloha + "/" + uuid));
+                String uri = ac.getUri().toString();
+                String basedOnVersionUri =
```

Comment:
Not very great that we depend on the URI.

Reply:
Will change back to creating the URI